d3a_api_client/setups/test_market_aggregator.py

```diff
@@ -5,6 +5,10 @@
     get_sim_id_and_domain_names
 from d3a_api_client.rest_market import RestMarketClient
 
+# import logging
+logger = logging.getLogger()
+logger.disabled = False
```

Comment:
Why are you disabling the logger?

Reply:
I added it in the first line. I will delete the commented lines in the code.